### PR TITLE
[ECO-118] Update the picker to be Hana compliant

### PR DESCRIPTION
**Pull request by micaelbergeron**
This PR re-skins the mural-picker to be aligned with the Hana theme.

### Screenshots (part of the sample app)

image

image

### CODE REVIEW

File: packages/mural-picker/src/components/mural-picker/styles.scss [hunk 1]
```diff
@@ -12,95 +12,55 @@
 
   .mural-picker-selects {
     display: flex;
-    justify-content: space-between;
-    align-items: stretch;
-    column-gap: 0.33em;
+    // justify-content: space-between;
```
Comment (KavenTheriault) [inline packages/mural-picker/src/components/mural-picker/styles.scss:15]: Comment to remove?

File: packages/mural-picker/src/components/mural-picker/index.tsx [hunk 2]
```diff
@@ -24,9 +24,37 @@ import Loading from '../loading';
...
 import { ReactSlot } from '../../common/react';
+import { BackButton, PrimaryButton, sizeProvider } from '../common';
+import { Box, FormControl, SvgIcon } from '@material-ui/core';
+import cx from 'classnames';
+import Measure from 'react-measure';
+
+import '@muraldevkit/mural-integrations-common/styles/common.scss';
+import './styles.scss';
+
+// @TECHDEBT — One we have the @tactivos/ds-icons library
```
Comment (KavenTheriault) [inline packages/mural-picker/src/components/mural-picker/index.tsx:37]: **Typo**: You mean `Once` I suppose

File: packages/mural-picker/src/components/card-list-item/styles.scss [hunk 3]
```diff
@@ -16,57 +18,88 @@
...
   &.small-card {
-    width: 141px;
-    height: 120px;
+    font-size: 0.66em;
+    width: 140px;
 
     .card-thumbnail {
-      width: 141px;
-      height: 85px;
+      // width: auto;
```
Comment (KavenTheriault) [inline packages/mural-picker/src/components/card-list-item/styles.scss:90]: Comment to remove?

File: packages/mural-picker/src/components/theme.ts [hunk 4]
```diff
@@ -2,31 +2,92 @@ import { createMuiTheme, ThemeOptions } from '@material-ui/core/styles';
...
 export type Preset = 'light' | 'dark';
 
-export const THEMES: Record<Preset, ThemeOptions> = {
-  light: {
-    palette: {
-      type: 'light',
-      text: { primary: '#585858' },
-      primary: {
-        main: '#FF0066',
+// #E8005A
```
Comment (KavenTheriault) [inline packages/mural-picker/src/components/theme.ts:5]: Comment to remove?

File: packages/mural-picker/src/components/workspace-picker/index.tsx [hunk 5]
```diff
@@ -92,18 +94,12 @@ export default class WorkspacePicker extends React.Component<WorkspacePickerProp
...
-        text: { primary: currentTheme === 'light' ? '#585858' : '#a7a7a7' },
-      },
-    });
+    const muiTheme = createTheme(theme || 'light');
 
     if (isLoading) {
       return (
         <ThemeProvider theme={muiTheme}>
-          <div className={`workspace-picker-body ${theme}`}>
+          <div className={`workspace-picker-body ${muiTheme?.palette?.type}`}>
```
Comment (KavenTheriault) [inline packages/mural-picker/src/components/workspace-picker/index.tsx:102]: Should we use `cx` here too?

File: packages/mural-picker/src/components/card-list/index.tsx [hunk 6]
```diff
@@ -1,28 +1,40 @@
...
+  const { firstName, lastName } = mural.createdBy;
 
-const dateMarker = (mural: Mural) => {
-  const span = Date.now() - mural.updatedOn;
-  const marker = humanize(span, { round: true, units: ['d', 'h'] });
+  return {
+    title: mural.title || 'Untitled mural',
+    details: dateMarkers(mural),
+    thumbnailUrl: mural.thumbnailUrl,
+    initials: firstName[0] || '' + lastName[0] || '',
```
Comment (KavenTheriault) [inline packages/mural-picker/src/components/card-list/index.tsx:16]: What if `firstName` or `lastName` are string empty or undefined ?

File: packages/mural-picker/src/components/mural-picker-form/index.tsx [hunk 7]
```diff
@@ -200,19 +201,12 @@ export default class MuralPickerForm extends React.Component<
...
-        text: { primary: currentTheme === 'light' ? '#585858' : '#a7a7a7' },
-      },
-    });
+    const muiTheme = createTheme(this.props.theme || 'light');
 
     return (
       <ThemeProvider theme={muiTheme}>
         <div
-          className={`mural-picker-body ${currentTheme}`}
+          className={`mural-picker-body ${muiTheme?.palette?.type}`}
```
Comment (KavenTheriault) [inline packages/mural-picker/src/components/mural-picker-form/index.tsx:209]: Use `cx` here?

File: packages/mural-picker/src/components/room-picker/index.tsx [hunk 8]
```diff
@@ -176,24 +178,11 @@ export default class RoomPicker extends React.Component<PropTypes> {
...
-      typography: {
-        fontFamily: 'Proxima Nova',
-      },
-    });
+    const muiTheme = createTheme(theme || 'light');
 
     return (
       <ThemeProvider theme={muiTheme}>
-        <div className={`room-picker-body ${theme}`}>
+        <div className={`room-picker-body ${muiTheme?.palette?.type}`}>
```
Comment (KavenTheriault) [inline packages/mural-picker/src/components/room-picker/index.tsx:185]: Use `cx` here?

**Review by KavenTheriault — COMMENTED** [review]
I left some comments. I would like to try it in MsTeams but didn't got time. I will do it later.

**Comment by micaelbergeron** [comment]
Thanks @KavenTheriault, addressed your comments, please take another look.

**Review by KavenTheriault — APPROVED** [review]
I tried it in MsTeams and it's working well.
LGTM

Peek 2022-10-31 13-42

File: packages/mural-picker/src/components/theme.ts [hunk 9]
```diff
@@ -2,31 +2,90 @@ import { createMuiTheme, ThemeOptions } from '@material-ui/core/styles';
...
+      root: {
+        padding: '0.8rem',
       },
     },
-    typography: {
-      fontFamily: 'Proxima Nova',
+    MuiButton: {
+      root: {
+        height: '2.8em',
+      },
```
Comment (msmolens) [inline packages/mural-picker/src/components/theme.ts:44]: question: Should the theme define a focus style for the buttons? Right now I don't see any visual indicator of the focus.

Comment (micaelbergeron) [reply packages/mural-picker/src/components/theme.ts:44]: I've updated the code to have a proper focus state now.

**Pull request merged**